interface/docker_build_interface.sh

```diff
@@ -0,0 +1,22 @@
+docker build -t adage-server/docker-interface-base -f Dockerfile.base .
```

Comment:
Well, it "rebuilds" it, but the way Docker works is that it if everything is already up to date and there are no changes, then it just uses the cache of the previously built layers, so it takes very little time (a couple of seconds), and does not actually build and store any new images of the container. However, this is a good way to check if anything has changed.